operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -115,6 +115,15 @@
           + "Defaults to true if `domainHomeSourceType` is PersistentVolume; false, otherwise.")
   private Boolean logHomeEnabled; // Boolean object, null if unspecified
 
+  /**
+   * Whether the port forwarding is enabled.
+   */
+  @Description(
+          "Specifies whether the port forwarding is enabled. "
+                  + "Defaults to true.")
+  private Boolean portForwardingEnabled = true;
```

Comment:
Need to add portForwardingEnabled to equals(), hashMap(), and toString()

Reply:
Done.